roxie/ccd/ccdmain.cpp

```diff
@@ -1100,14 +1100,15 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
             maxTotalMemoryLimit = friendlyStringToSize(resourcedMemory);
             maxTotalMemoryLimit = maxTotalMemoryLimit / 100.0 * roxieMemResourcedMemoryPct;
         }
+        bool lockMemory = topology->getPropBool("@heapLockMemory", false);
```

Comment:
I would be inclined to default to true

Reply:
(May also need to default retainMemory too, given the code in roxiemem...)

Reply:
ok.
Perhaps some OS/shell limits on the amount of lockable memory might cause some installations to fail the locking ?, but things should continue on ok except for the log message about failed to lock heap ...

Reply:
Still saying false...

Reply:
updated

Reply:
ok, so false for 8.6.x and then soon after reset 8.8 default to tue.